1-exercises/A-undefined/exercise.js

```diff
@@ -12,25 +12,28 @@
 // Example 1
 let a;
 console.log(a);
+// a hasn't been initialized, it has no value and that is way you see undefined
 
 
 // Example 2
 function sayHello() {
     let message = "Hello";
 }
+// there is no action, you've just initialized message but your function does nothing else 
 
 let hello = sayHello();
 console.log(hello);
-
+// the function you're using to initialize hello does not return anything so hello is not initialized
 
 // Example 3
 function sayHelloToUser(user) {
     console.log(`Hello ${user}`);
 }
 
 sayHelloToUser();
-
```

Comment:
The variable is initialised to the default value of "undefined". But , I agree, we have not explicitly initialised a variable to be fed as an argument to the function sayHelloToUser(user)